hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/fsck/ContainerHealthStatus.java

```diff
@@ -78,6 +100,14 @@ public int getReplicationFactor() {
   }
 
   public boolean isHealthy() {
+    return containerReplicaCount.isHealthy();
```

Comment:
This is changing the meaning here. RepCount isHealthy is checking for the container being closed / quasi-closed and all replicas match that. The original isHealthy here was about not under / over rep and also not mis-rep.

Reply:
I know, that's why I renamed the original method to `isProperlyReplicated()` and now every method in Recon, calling `isHealthy()`, now calls `isProperlyReplicated()`.

It's part of the problem, that Recon uses the same method naming as the SCM but has a different meaning for most of them.